src/watertap_contrib/seto/costing/solar/trough.py

```diff
@@ -0,0 +1,161 @@
+import pyomo.environ as pyo
+from watertap.costing.util import register_costing_parameter_block
+from watertap_contrib.seto.costing.util import (
+    make_capital_cost_var,
+    make_fixed_operating_cost_var,
+    make_variable_operating_cost_var,
+)
+
+
+def build_trough_cost_param_block(blk):
+
+    costing = blk.parent_block()
+
+    blk.base_storage_hours = pyo.Var(
+        initialize=6,
+        units=pyo.units.hour,
+        bounds=(6, 6),
```

Comment:
I would remove these bounds, or if the intention is to (almost) never change this, you could either (1) make this a mutable parameter or (2) keep this variable declaration and fix it to the value of 6 right after.

Reply:
IMHO - it makes more sense to just make this a mutable Parameter.

Reply:
Thanks, I'll change to a mutable Parameter.